modules/apps/frontend-data-set/frontend-data-set-admin-web/src/main/resources/META-INF/resources/js/data_set/visualization_modes/modes/Table.tsx

```diff
@@ -116,10 +116,19 @@ const NewFieldModalContent = ({
 						{Liferay.Language.get('field-name')}
 
 						<RequiredMark />
+
```

Comment:
Is there a reason to not use the `AddCustomFieldModalContent.tsx` component from the table visualization mode?

Reply:
Good question, seems like duplicated code.

Reply:
This task seems like a good opportunity to fix this on the way, or reach a conclusion why not to do it. It is a low priority task.

Reply:
Unfortunately, it is likely that we will _never_ be able to have a task dedicated just to remove duplicated code.

Reply:
Guys, I am very happy to find things to improve, but, we only included this task as a minor improvement that we could do while we were not able to open the next epic. Now we have been digging into testing and now a refactor, and we have stories way more urgent and valuable to do. Either we finish it as it is, or we can pospone this to focus on the milestone commitments.

Reply:
I understand your concern Gerardo, and of this would start to become complicated, and cause days of delays, I'd agree with you. But, unless I'm missing something, this is a tiny code improvement that can be fixed in few minutes.